### PR TITLE
Fix spelling errors and broken links

### PR DESCRIPTION
**Pull request by DarthShmev**
Bounty complete for https://www.triplechecker.com/reports?website=tenancyforlaravel-com. darthshmev#8107 on discord.

### CODE REVIEW

File: source/docs/v3/early-identification.blade.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ class PostController
 }
 ```
 
-If you don't like that because you access some dependency from many actions, consider creating a memoized method:
```
Comment (stancl) [inline source/docs/v3/early-identification.blade.md:36]: `memoized` is correct here

File: source/docs/v3/introduction.blade.md [hunk 2]
```diff
@@ -33,4 +33,4 @@ In the current version, we're also making [manual tenancy]({{ $page->link('manua
 
 ## Tenant identification {#tenant-identification}
 
-For your application to be tenant-aware, a [tenant has to be identified]({{ $page->link('tenant-identification') }}). This package ships with a large number of identification middlewares. You may identify tenants by domain, subdomain, domain OR subdomain at the same time, path or request data.
```
Comment (stancl) [inline source/docs/v3/introduction.blade.md:36]: Change this to `This package ships with many identification middleware classes.` please.

`middleware` is technically the correct plural but it isn't as clear.

**Review by stancl — CHANGES_REQUESTED** [review]
Hi,

Thanks a lot. A few changes. Please fix these and I'll immediately merge & pay.

File: source/docs/v3/early-identification.blade.md (hunk 1 above)
Comment (DarthShmev) [reply source/docs/v3/early-identification.blade.md]: Done.

File: source/docs/v3/introduction.blade.md (hunk 2 above)
Comment (DarthShmev) [reply source/docs/v3/introduction.blade.md:36]: Done.

**Comment by stancl** [comment]
Thanks!

**Pull request merged**